app/views/hyrax/content_blocks/_form.html.erb

```diff
@@ -0,0 +1,96 @@
+<div class="panel panel-default tabs">
+  <ul class="nav nav-tabs" role="tablist">
+    <li class="active">
+      <a href="#about_page" role="tab" data-toggle="tab"><%= t(:'hyrax.content_blocks.tabs.about_page') %></a>
+    </li>
+    <li>
+      <a href="#help_page" role="tab" data-toggle="tab"><%= t(:'hyrax.content_blocks.tabs.help_page') %></a>
+    </li>
+    <li>
+      <a href="#announcement_text" role="tab" data-toggle="tab"><%= t(:'hyrax.content_blocks.tabs.announcement_text') %></a>
+    </li>
+    <li>
+      <a href="#marketing_text" role="tab" data-toggle="tab"><%= t(:'hyrax.content_blocks.tabs.marketing_text') %></a>
+    </li>
+    <li>
+      <a href="#featured_researcher" role="tab" data-toggle="tab"><%= t(:'hyrax.content_blocks.tabs.featured_researcher') %></a>
+    </li>
+  </ul>
+  <div class="tab-content">
+    <div id="about_page" class="tab-pane active">
+      <div class="panel panel-default labels">
+        <%= simple_form_for ContentBlock.about_page, url: hyrax.content_block_path(ContentBlock.about_page) do |f| %>
+          <div class="panel-body">
+            <div class="field form-group">
+                <%= f.text_area :about_page, value: f.object.value, class: 'form-control tinymce', rows: 20, cols: 120 %>
```

Comment:
I'm not sure what to name the text area here. (Related to comment above about permitted params in the controller.)